### ISSUE TITLE
allow '**' in ignore files 

### ISSUE THREAD

**Issue by broeder-j** [aiidateam/aiida-testing#25]
Feature request, nice to have.

It would be very nice if one could allow for bash regs in the ignorefile list.
example 
```
ignore_files=['cdn**'] # to ignore files with names cdn01, cdn02, cdn23, ...
```
(if a code produces an arbitray number of files with certain naming pattern)

Otherwise thanks for your work! Currently everything is working nicely on linux. On mac I had an issue, which I did not yet managed to track down yet.

**Comment by greschd** [comment]
Should be easy enough to do, using the [fnmatch.fnmatchcase](https://docs.python.org/3.5/library/fnmatch.html#fnmatch.fnmatchcase) to match filenames here: https://github.com/aiidateam/aiida-testing/blob/develop/aiida_testing/mock_code/_cli.py#L54

Do you want to do this?

**Comment by broeder-j** [comment]
I can do it.

**Comment by greschd** [comment]
Thanks! Feel free to open a separate issue for the mac problem - maybe it's to do with the use of environment variables to pass arguments to the executable?

**Comment by greschd** [comment]
Closed in #26.
